src/bootstrap-input-spinner.js

```diff
@@ -96,6 +104,9 @@
             $original[0].setValue = function (newValue) {
                 setValue(newValue)
             }
+            $original[0].destroy = function () {
```

Comment:
Adding a `destroy` function to the original javascript object would be fine by me, but I would make it more verbose that it does not actually destroys the element, but rather its associated spinner. Therefore, here's my suggestion :

```suggestion
            $original[0].destroySpinner = function () {
```

Of course the call to it at your line 27 would change to :
```js
setTimeout(function () {
    element.destroySpinner()
})
```

That's just my two cents!

Reply:
sounds good, feel free to make the change.

Reply:
Yes, great, it should return ti the exact state before creation. But I am uncertain about the syntax. Could be `element.destroyInputSpinner()`, `element.inputSpinner("destroy")` or `element.inputSpinner.destroy()`, or something I missed. What is the best practice here?

Reply:
i agree the string method is nicer.